ert3/workspace/_workspace.py

```diff
@@ -24,6 +24,17 @@ def assert_experiment_exists(workspace_root, experiment_name):
         )
 
 
+def get_experiments(workspace_root):
+    experiment_root = Path(workspace_root) / ert3.workspace.EXPERIMENTS_BASE
+    if not experiment_root.exists():
```

Comment:
I find this to be too forgiving. I would suggest that we instead raise an `IllegalWorkspaceState` as a new exception introduced in `ert3.exceptions`. Maybe we want to allow for workspaces without any experiments? But let us wait with allowing that until we have a clear strategy for it :)